x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/helpers.ts

```diff
@@ -24,13 +24,13 @@ import { ESQuery } from '../../../../../common/typed_json';
  */
 export const isNoisy = (hits: number, timeframe: Unit): boolean => {
   if (timeframe === 'h') {
-    return hits > 1;
+    return hits > 20;
```

Comment:
was there a particular reason for this change? as far as I know, the noisiness criteria was 1 alert per hour for all scenarios

Reply:
Ah maybe i misunderstood the code prior to then, I thought we were going based of the invocations themselves, we can switch that back and get a more solidified answer for noisiness in all rule scenarios